lib/puppet/type/user.rb

```diff
@@ -423,7 +423,10 @@ def insync?(current)
         resource at the same time. For instance, Puppet creates a home directory for a managed
         user if `ensure => present` and the user does not exist at the time of the Puppet run.
         If the home directory is then deleted manually, Puppet will not recreate it on the next
-        run."
+        run.
+
+        Note that on Windows, this manages creation/deletion of the user profile instead of the
```

Comment:
Are you referring to the user profile dir? or the user itself?

This makes it sound like the without managehome you can't manage the user at all.

Reply:
Yeah, I see where you're coming from. It's referring to the user profile directory (what's in `C:\Users\<username>`. Let me reword that

Reply:
Actually, I wonder if changing the wording is necessary. We do create the user profile [here](https://github.com/puppetlabs/puppet/blob/master/lib/puppet/util/windows/user.rb#L109), there is a separate [WMI class](https://msdn.microsoft.com/en-us/library/ee886409(v=vs.85).aspx) for it, and we delete the profile by deleting the corresponding instance of that class [here](https://github.com/puppetlabs/puppet/blob/master/lib/puppet/util/windows/adsi.rb#L496). `user profile` is also a term that Windows sysadmins will likely know, and the user profile is more than just the `C:\Users\<username>` directory so user profile directory is a bit misleading (especially when it can be moved). Thus, I think the wording is fine given all of this.

Reply:
Updated the docs. Are they clearer now?

Reply:
Personally I still think the use of the words `user profile` indicates that managehome is somehow toggling something about the user as a whole, rather than just this directory.

However I don't feel strongly enough about it to raise any further as a disagreement. If the second reviewer approves the update so do I.

Reply:
Also, we may want to link to some MS docs about profile dirs and what they are for etc. maybe https://msdn.microsoft.com/en-us/library/windows/desktop/bb776896(v=vs.85).aspx ?

Just a thought

Reply:
Yeah, a link would be useful. Do we have any policies for posting links in our docs? There could be some concerns about that link possibly changing in the future.

Reply:
Ah, I don't know if there's any policies.

Reply:
So as someone unfamiliar with Windows administration, I read this as: A user profile is not created unless managehome is true. If managhome is true, it will create the profile and store it in, typically, C:\Users\<username>. Is that correct?

Reply:
@caseywilliams Almost -- the `typically` is `always`. Updated the PR to make this explicit.